source/includes/webhooks/_candidate_events.md

```diff
@@ -1202,3 +1202,66 @@ See web hook [common attributes](#common-attributes).
+
+```json
+{
+  "action": "candidate_anonymized",
+  "payload": {
+    "candidate_id": 37031511,
+    "anonymized_fields": ["full_name", "current_title"]
+  }
+}
+```
+
+### Anonymized Fields
+
+A list of possible anonymized fields are provided. If the item in "attribute" is received in the anonymized_fields array, the corresponding changes in notes have been applied in Greenhouse. Note that some of these fields may not be accessible via other web hooks or via Harvest.
+
+| Attribute | Anonymize Process |
+|------------|--------|
+| `activity_items` | Destroy all items in the candidate's activity feed. |
+| `addresses` | Destroy all values from `candidate.addresses` |
+| `all_offer_versions` | For each of this candidate's applications, destroy all offers. (For legacy reasons, this will also raise an [Offer Deleted](#offer-deleted) web hook if one is configured.) |
+| `application_questions` | For each of this candidate's applications, destroy all answers to all application questions. |
+| `attachments` | Destroy all attachments on this candidate and all their associated applications. |
+| `candidate_photo` | Destroy the candidate's photo. |
+| `candidate_stage_data` | For each of this candidate's applications, set the `created_at` time to now, remove the candidate from the current stage, destroy all stage transition information, and set the candidate back to the first stage. |
+| `coordinator` | Set `candidate.coordinator` to null |
+| `credited_to` | For each of this candidate's application, destroy `application.credited_to`. |
+| `custom_application_fields` | For each of this candidate's applications, destroy `application.custom_fields`. |
+| `custom_candidate_fields` | Destroy all values from `candidate.custom_fields` |
+| `custom_rejection_question_fields` | For each of this candidate's application, destroy all custom fields on rejection details. |
+| `current_company` | Set `candidate.company` to null. |
+| `current_title` | Set `candidate.title` to null. |
+| `education` | Remove all values from `candidate.educations` |
+| `email_addresses` | Destroy all values from `candidate.email_addresses` |
+| `employment` | Remove all values from `candidate.employments` |
+| `emails` | Destroy record of any e-mails sent to this candidate via Greenhouse. |
+| `follow_up_reminders` | Destroy any follow-up reminders configured on this person. |
+| `full_name` | Set `candidate.first_name` to "Anonymized" and `candidate.last_name` to the `candidate.id`. |
+| `headline` | Set the candidate's headline to null. |
+| `inmails` | Destroy record of any LinkedIn Inmails sent to this candidate via Greenhouse. |
```

Comment:
The InMails are technically coming from LinkedIn, so it might be more correct to say 

`Destroy record of any LinkedIn InMails synced to this candidate from LinkedIn.`